packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -2091,6 +2091,149 @@ HTTP/1.1 200 OK
+
+Passing an invalid `brandId`, or `templateName` returns a `404 Not Found` with error code `E0000007`.
+
+#### Use examples
+
+The following example returns the settings for the email template `UserActivation`.
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+'https://${yourOktaDomain}/api/v1/brands/${brandId}/templates/email/UserActivation/settings'
+```
+
+##### Response
+
+```http
+HTTP/1.1 200 OK
+```
+
+```json
+{
+    "recipients": "ALL_USERS",
+    "_links": {
+      "template": {
+            "href": "https://${yourOktaDomain}/api/v1/brands/${brand}/templates/email/UserActivation",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/brands/${brand}/templates/email/UserActivation/settings",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Update Email Template Settings
+
+<ApiOperation method="put" url="/api/v1/brands/${brandId}/templates/email/${templateName}/settings" />
+
+Updates the settings associated with the template
+
+#### Request path parameters
+
+| Parameter         | Type        | Description                  |
+| ----------------- | ----------- | ---------------------------- |
+| `brandId`         | String      | ID of a Brand                |
+| `templateName`    | String      | Name of an Email Template    |
+
+#### Request body
+
+The [Email Template Settings Object](#email-template-settings-object) resource to update.
+
+#### Response body
+
+The [Email Template Settings Object](#email-template-settings-object) resource that was updated.
+
+Returns a `422 Unprocessable Entity` if the template doesn't support the recipient's value.
+
+> **NOTE**: The templates that do not support all recipient values are New Sign-On Notification, Authenticator Enrolled, Authenticator Reset, Password Changed and the Change Email Confirmation templates.
+
+Passing an invalid `brandId` or `templateName` returns a `404 Not Found` with error code `E0000007`.
+
+#### Use examples
+
+The following example disables the `UserActivation` email from being sent.
+
+##### Request
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "recipients": "NO_USERS",
+}' \
+'https://${yourOktaDomain}/api/v1/brands/${brandId}/templates/email/UserActivation/settings'
+```
+
+##### Response
+
+```http
+HTTP/1.1 200 OK
+```
+
+```json
+{
+    "recipients": "NO_USERS",
+    "_links": {
+        "template": {
+            "href": "https://${yourOktaDomain}/api/v1/brands/${brand}/templates/email/UserActivation",
```

Comment:
maybe use the name ${brandId} in the links as well to keep it consistent with the request url on 2203?

Reply:
@chrisclare-okta what do you think?

Reply:
Yup, this is correct. I'll change that now